tests/Collection/ArrayCollectionIsImmutableExceptionTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Arrays\Tests\Collection;
+
+use PHPUnit\Framework\TestCase;
+use Yiisoft\Arrays\Collection\ArrayCollectionIsImmutableException;
+
+final class ArrayCollectionIsImmutableExceptionTest extends TestCase
+{
+    public function testMessage(): void
+    {
+        $this->expectExceptionMessage('ArrayCollection is immutable object.');
+        throw new ArrayCollectionIsImmutableException();
```

Comment:
Does it really make sense to test it?

Reply:
> Does it really make sense to test it?

for code coverage indicator :)

Reply:
Maybe it's a better idea to test the real case when this exception is thrown?